### PR TITLE
Change sawtooth batch-status to sawtooth batch status cli

### PR DESCRIPTION
**Pull request by feihujiang**
Signed-off-by: feihujiang <jiangfeihu@huawei.com>

As discussed in the Sawtooth Channel, the sawtooth batch status is much more consistent than sawtooth batch-status. So this PR changed sawtooth batch-status to sawtooth batch status cli, and added docs for it.

### CODE REVIEW

File: bin/generate_cli_output [hunk 1]
```diff
@@ -22,6 +22,7 @@ save_usage sawtooth admin keygen
 save_usage sawtooth batch
 save_usage sawtooth batch list
 save_usage sawtooth batch show
+save_usage sawtooth batch-status
```
Comment (jjason) [inline bin/generate_cli_output]: I missed the PR that added this sub-command.  As far as the documentation goes, I have no problem with it.  However, I started a discussion on RocketChat about whether or not this is the appropriate name for the sub-command.

File: docs/source/cli/sawtooth.rst [hunk 2]
```diff
@@ -112,6 +112,13 @@ sawtooth batch show
    :language: console
    :linenos:
 
+sawtooth batch-status
+===================
```
Comment (jsmitchell) [inline docs/source/cli/sawtooth.rst]: The doc build is breaking on this line (the equals underline needs to be as long as the title on the previous line).

Comment (feihujiang) [inline docs/source/cli/sawtooth.rst]: Done

**Comment by feihujiang** [comment]
As discussed in the Sawtooth Channel,  the ```sawtooth batch status``` is much more consistent than ```sawtooth batch-status```. So this PR changed sawtooth batch-status to sawtooth batch status cli, and added docs for it.

**Comment by feihujiang** [comment]
@jsmitchell I updated this PR, please take a look. Thanks.

**Comment by jjason** [comment]
Tested locally

**Pull request merged**